libraries/networking/src/AddressManager.cpp

```diff
@@ -606,7 +633,7 @@ void AddressManager::handleAPIError(QNetworkReply* errorReply) {
 
 void AddressManager::attemptPlaceNameLookup(const QString& lookupString, const QString& overridePath, LookupTrigger trigger) {
     // assume this is a place name and see if we can get any info on it
-    QString placeName = QUrl::toPercentEncoding(lookupString);
+    //QString placeName = QUrl::toPercentEncoding(lookupString);
```

Comment:
Commented out code should be removed

Reply:
Fixed. I'm not sure if I need to do percentEncoding anymore since I caught the only case I can think of where that would be relevant... Oh well. We'll see.